functions/misc/server.py

```diff
@@ -67,6 +67,17 @@ def get_memory_usage():
         print("error getting memory usage")
         os._exit(2)
 
+def get_ip_address_from_env():
+    try:
+        machine_ip = "unset"
+        if "HOST_MACHINE_IP" in os.environ.keys():
+            machine_ip = os.environ["HOST_MACHINE_IP"]
+        return machine_ip
```

Comment:
wouldn't `os.getenv("HOST_MACHINE_IP", "unset")`  be cleaner?

Reply:
Actually come to think of it it might be better to get rid of the entire get_ip_address_from_env(): function as all it does is basically get an envvar with a default value and have it passed form the parse_it parser in the main `worker.py` with all the rest of the config variables